### PR TITLE
Documenting SQL Models

### PR DESCRIPTION
**Pull request by atbida**

### CODE REVIEW

**Review by several27 — CHANGES_REQUESTED** [review]

**Comment by atbida** [comment]
@several27 please take a look to see if i've sufficiently covered the DBT topics while keeping this page not-too-long. Meanwhile I'll move onto the rest of the SQL docs.

File: docs/concepts/models/models.md [hunk 1]
```diff
@@ -1,13 +1,69 @@
...
+
+Data models incorporate the step-by-step logic to transform raw data to some intermediate or final state. Each Model, stored as a `.sql` file on Git, is managed as software - with best practices like peer review and version control. The Model can include Common Table Expressions (CTEs) and [refer](https://docs.getdbt.com/docs/build/sql-models#building-dependencies-between-models) to other Models. Importantly, SQL statements with Prophecy and dbt are re-usable. When a Model is updated, any reference to that Model is likewise updated.
+
+Here we explore how to use Models in Prophecy, adopting the concept and vernacular from dbt Core™. Later, we'll see how to import dbt projects to Prophecy, or you can create a new Project and Models using Prophecy's drag-and-drop interface. After you've read this page, get hands on with Models in this getting-started [guide](/docs/getting-started/getting-started-with-low-code-sql.md#44-Develop-your-first-model).
+
+### Using Models in Prophecy
+
+Prophecy displays Models using a lineage view, a visual view, and a code view.
```
Comment (atbida) [inline docs/concepts/models/models.md:47]: show: add a model

Comment (atbida) [reply docs/concepts/models/models.md:47]: we show this in the getting started doc, so I'll not show here.

**Review by several27 — APPROVED** [review]

File: docs/concepts/models/models.md [hunk 2]
```diff
@@ -1,13 +1,69 @@
...
+
+In Prophecy and dbt, **Data Models** are SQL statements that build a single table or view - and allow for better management.
+
+Data models incorporate the step-by-step logic to transform raw data to some intermediate or final state. Each Model, stored as a `.sql` file on Git, is managed as software - with best practices like peer review and version control. The Model can include Common Table Expressions (CTEs) and [refer](https://docs.getdbt.com/docs/build/sql-models#building-dependencies-between-models) to other Models. Importantly, SQL statements with Prophecy and dbt are re-usable. When a Model is updated, any reference to that Model is likewise updated.
+
+Here we explore how to use Models in Prophecy, adopting the concept and vernacular from dbt Core™. Later, we'll see how to import dbt projects to Prophecy, or you can create a new Project and Models using Prophecy's drag-and-drop interface. After you've read this page, get hands on with Models in this getting-started [guide](/docs/getting-started/getting-started-with-low-code-sql.md#44-Develop-your-first-model).
+
+### Using Models in Prophecy
```
Comment (bandikishores) [inline docs/concepts/models/models.md:45]: Should we add a section for Macros as well?

**Review by bandikishores — APPROVED** [review]

File: docs/concepts/models/models.md (hunk 2 above)
Comment (atbida) [reply docs/concepts/models/models.md:45]: Yes, this will be part of more documentation planned for the low-code-sql docs section.

**Pull request merged**